src/main/java/fr/iut/pathpilotapi/auth/exceptions/GlobalExceptionHandler.java

```diff
@@ -58,6 +58,16 @@ public ProblemDetail handleSecurityException(Exception exception) {
             errorDetail.setProperty(DESCRIPTION, "The JWT token has expired");
         }
 
+        if (exception instanceof EmailAlreadyTakenException) {
+            errorDetail = ProblemDetail.forStatusAndDetail(HttpStatusCode.valueOf(409), exception.getMessage());
```

Comment:
I think a better http error code can be chosen.
Because `409` show a 

> The HTTP 409 Conflict status code indicates a request conflict with the current state of the target resource.

Here the issue is that the email already exist